docs/src/main/sphinx/release/release-470.md

```diff
@@ -0,0 +1,125 @@
+# Release 470 (5 Feb 2025)
+
+## General
+
+* Add [](/connector/duckdb). ({issue}`18031`)
+* Add Loki connector. ({issue}`23053`)
+* Add support for access control systems to control privileges for
```

Comment:
@martint @homar @dain .. reworded original proposal .. I am not sure if the scope is complete or if other statements are also impacted

Reply:
my bad, I don't think release notes are required for that commit, sorry for the confusion

Reply:
What do you think @dain and @martint .. do we want an entry here .. or in the SPI section. or not at all?